.travis.yml

```diff
@@ -0,0 +1,24 @@
+language: node_js
+
+sudo: required
+
+node_js:
+ - "0.10"
+ - "4"
+
+branches:
+  only:
+    - develop
+    - master
+
+services:
+ - docker
+ - mongodb
```

Comment:
You're fine with running your tests against a 2.6 (I think it is) version of MongoDB, right? Since that it the one currently Travis would use according to https://github.com/travis-ci/travis-ci/issues/3694 :)